protostar/commands/test/testing_summary.py

```diff
@@ -96,7 +103,7 @@ def _get_test_suites_summary(self, collected_test_suites_count: int) -> str:
         for suit_case_results in self.test_suites_mapping.values():
             partial_summary = TestingSummary(suit_case_results, self.testing_seed)
 
-            if len(partial_summary.broken) > 0:
+            if len(partial_summary.broken_suites) + len(partial_summary.broken) > 0:
```

Comment:
```suggestion
            if len(partial_summary.broken_suites) > 0:
```
Consider use case: 10 broken test cases in a one test suite